test/vm-run

```diff
@@ -48,6 +49,24 @@ NETWORK_SCRIPT="""
   <ip family="ipv6" address="fd00:111:112::1" prefix="64"/>
 </network>
 EOF
+
+    if [ ! -u /usr/libexec/qemu-bridge-helper ]; then
+        chmod -v u+s /usr/libexec/qemu-bridge-helper
+    fi
+
+    qemu_config_dirs=()
+    for d in /etc/qemu-kvm/ /etc/qemu; do
+       if test -f ${d}/bridge.conf; then
+           qemu_config_dirs+=(${d})
```

Comment:
Arrays are a bashism, and the script runs through `/bin/sh` only. So if you really want to keep the array syntax, please change the invocation of the script to `bash`. However, for this simple case you can just keep a simple string variable (`qemu_config_dirs="$qemu_config_dirs $d`) and iterate through it with normal shell word splitting (`for qemu_config_dir in $qemu_config_dirs`) which makes things a little more shell-ish to read.